eyeshade/migrations/0005_account_types/down.sql

```diff
@@ -0,0 +1,95 @@
+
+
+
+
+drop materialized view account_balances;
+drop view account_transactions;
+
+alter table transactions drop constraint check_from_account_type;
+alter table transactions drop constraint check_to_account_type;
+alter table transactions drop constraint check_transaction_type;
+
+alter table transactions add constraint check_from_account_type
+  check (from_account_type in ('channel', 'owner', 'uphold', 'internal', 'payment_id'));
+alter table transactions add constraint check_to_account_type
+  check (to_account_type in ('channel', 'owner', 'uphold', 'internal', 'payment_id'));
+
+alter table transactions add constraint check_transaction_type
+  check (transaction_type in ('contribution', 'referral', 'contribution_settlement', 'referral_settlement', 'fees', 'scaleup', 'manual', 'ad', 'ad_settlement', 'manual_settlement'));
+
+create view account_transactions(
```

Comment:
I'm confused about this - we didn't delete the view in `up.sql`. also shouldn't this file have a guard per your other recent changes to the `down.sql` files?

Reply:
yes. looks like i didn't push that change.